### ISSUE TITLE
Export Library as iTunes XML

### ISSUE THREAD

**Issue by dashbad** [jcm93/jmc#30]
Hi,

Love what you have started here. I have a feature request.

It would be awesome if this player could create and place (automatically even better) a spoof itunes library.xml in ~/Music/iTunes based on your jmc library, including all regular and smart playlists.

Reason being that quite a lot of third party software - notably DJ software like Rekordbox, Serato and Traktor & Music production software like Logic and Garageband, integrate with iTunes using this file. This integration has prevented me from migrating to a third party player on OSX and it would be great to be able to ditch iTunes.

Thanks

**Comment by colin-marshall** [comment]
This would be a great feature. The Beatport app has it.

**Comment by jcm93** [comment]
Good idea. I'll put this on the docket. Library exports in general are something I need to tackle, and supporting the iTunes XML format would indeed be useful.

Side note, development is on pause at the moment; my laptop is broken and in the shop. Should be back by end of week however.

**Comment by Porco-Rosso** [comment]
As a further bonus, an iTunes XML export, would potentially allow you to use itunes to sync your playlists created in jmc to iDevices.
It's a workaround, but I know syncing woes are a concern for many who wish to ditch itunes. 

_Note: I think by default iTunes loads from the .itl file and the .xml only serves as backup?_
